cypress.json

```
@@ -2,6 +2,7 @@
   "baseUrl": "http://localhost:8080",
   "pluginsFile": "tests/e2e/plugins/index.js",
   "projectId": "gfk5oj",
+  "graphqlTimeout": 20000,
```

```suggestion
  "graphqlTimeout": 10000,
```
I would try to reduce it to 10s. As it's so easy to change, we can increase it if necessary, but let's give it a try.